deps/graph-parser/src/logseq/graph_parser/util/db.cljs

```diff
@@ -22,56 +22,93 @@ it will return 1622433600000, which is equivalent to Mon May 31 2021 00 :00:00."
   (parse-long
    (string/replace (date-time-util/ymd date) "/" "")))
 
+(defn old->new-relative-date-format [input]
+  (let [count (re-find #"^\d+" (name input))
+        plus-minus (if (re-find #"after" (name input)) "+" "-")
+        ms? (string/ends-with? (name input) "-ms")]
+    (keyword :today (str plus-minus count "d" (if ms? "-ms" "")))))
+
+(comment
+  (old->new-relative-date-format "1d")
+  (old->new-relative-date-format "1d-before")
+  (old->new-relative-date-format "1d-after")
+  (old->new-relative-date-format "1d-before-ms")
+  (old->new-relative-date-format "1d-after-ms")
+  (old->new-relative-date-format "1w-after-ms"))
+
+(defn keyword-input-dispatch [input]
+  (cond 
+    (#{:current-page :current-block :parent-block :today :yesterday :tomorrow :right-now-ms} input) input
+    (re-find #"^[+-]\d+[dwmy](-ms)?$" (name input)) :relative-date
+    (re-find #"^\d+d(-before|-after|-before-ms|-after-ms)?$" (name input)) :DEPRECATED-relative-date
+    (= :start-of-today-ms input) :DEPRECATED-start-of-today-ms 
+    (= :end-of-today-ms input) :DEPRECATED-end-of-today-ms))
+(defmulti resolve-keyword-input (fn [_db input _opts] (keyword-input-dispatch input))) 
+
+(defmethod resolve-keyword-input :DEPRECATED-relative-date [db input opts]
+  ;; This handles all of the cases covered by the following:
+  ;; :Xd, :Xd-before, :Xd-before-ms, :Xd-after, :Xd-after-ms
+  (resolve-keyword-input db (old->new-relative-date-format input) opts))
+
+(defmethod resolve-keyword-input :DEPRECATED-start-of-today-ms [db _ opts]
+  (resolve-keyword-input db :today/-0d-ms opts))
```

Comment:
I don't know why -0d-ms and +0d-ms aren't intuitive to me. Maybe it's worth writing these in hour form or not deprecating these two?

Reply:
Yeah I didn't know what to do about these keywords. Since the logic is the same as `:today/-0d-ms`, I thought it best to keep the logic in one place instead of just duplicate it. 

Maybe deprecated isn't the right word, ALIASED? The only reason to indicate anything is 'performance' as this input will re-resolve (I don't think resolving a single input though is going to be a real performance hit). Maybe I can remove any label?

Reply:
If we're not meaning to deprecate (to devs), then removing or ALIASED makes more sense

Reply:
I took these out of deprecated, now they're all going to a `:today-time` resolver :)